lnwallet/wallet.go

```diff
@@ -795,13 +802,15 @@ func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg
 	// when the PSBT has been verified.
 	isPublic := req.Flags&lnwire.FFAnnounceChannel != 0
 	hasAnchors := req.CommitType == CommitmentTypeAnchorsZeroFeeHtlcTx
-	err = l.enforceNewReservedValue(fundingIntent, isPublic, hasAnchors)
-	if err != nil {
-		fundingIntent.Cancel()
+	if enforceNewReservedValue {
```

Comment:
So the comment above about PSBTs was incorrect before this change?

Reply:
Yes.